fdbserver/BackupWorker.actor.cpp

```diff
@@ -78,8 +78,8 @@ struct BackupData {
 			container = IBackupContainer::openContainer("file://simfdb/mutation_backups/");
 		} else {
 			// TODO: use blobstore URL passed from somewhere.
-			ASSERT(false);
```

Comment:
The crash seems to be expected. 
It can be an intentional crash to alert operators as early as possible and make sure the worker won't keep running with an unhandled situation.

Did I miss something?

Reply:
Right, the crash was intended to alert operators. Since this is in the master branch now, the crash should be fixed so that there won't be crash in real deployment.

The fix is safe. Because right now the backup worker only runs in NOOP mode, there is no files been written out.

Reply:
Hmm, since this is in the master branch, won't it help us remember to fix the problem by intentionally crashing it?

I'm fine with either way.